### ISSUE TITLE
About the compression factor 17.2366666

### ISSUE THREAD

**Issue by Hellwz** [stefan-baumann/inceptionkeynet#1]
Thanks for sharing this code, really a good work.
I'm confused with this factor 17.2366666. From my caculation, the original frame rate is 44100 / 512 = 86.1328125. Then the factor equals 86.1328125 / 5= 17.2265625. I don't know if my calculation method is correct.
Looking forward to your reply.

**Comment by stefan-baumann** [comment]
Hey, thank you very much for bringing this up, great catch!
Your calculation should be correct and indeed, I cannot find an adequate explanation as to how I ended up with that discrepancy in the default factor. I have been looking into this since yesterday, but I have not gotten further than [this comment here](https://github.com/stefan-baumann/inceptionkeynet/blob/9c2bdc6e6e7c1f0083a72d960d0ff8b94ba57dca/inceptionkeynet/tasks/train.py#L47), where the comma decimal separator in the `86,183333333` indicates that I copied & pasted this from a calculator (when I type decimal numbers myself, I use a dot as the decimal separator), but I cannot figure out how I arrived at this exact value.
The best explanation I can offer right now is that I must have somehow mistyped when calculating this value. I'll continue to look into my documentation and the original repository (this repository is a subset of a larger repository that contains the code relevant to the paper) and see if I can find out anything else.

Luckily, this discrepancy results in an almost negligible speed difference (less than 0.1%), so this should at least not invalidate any results. But unless I can trace back my steps there and find out how and why I arrived at this specific, seemingly ever-so-slightly incorrect value, this is probably something I should fix.

Thanks again for pointing this out!
